client/src/components/support/StaffDashboard.tsx

```diff
@@ -411,14 +433,16 @@ const StaffDashboard = () => {
                     <TicketStatusLabel status={ticket.status} />
                   </Table.Cell>
                   <Table.Cell>
-                    <Button
-                      color="blue"
-                      size="tiny"
-                      onClick={() => openTicket(ticket.uuid)}
-                    >
-                      <Icon name="eye" />
-                      View
-                    </Button>
+                    <Link to={`/support/ticket/${ticket.uuid}`} target="_blank">
+                      <Button
+                        color="blue"
+                        size="tiny"
+                        onClick={() => openTicket(ticket.uuid)}
```

Comment:
`<Link>` but also a `<Button>` with `onClick`? maybe just use `as={Link}` if you want the styling